Trustroots/trustroots
# Offer Location in React

#### Proposed Changes

* Rewrite tr-offer-location in React. (Map in user profile offer.)

#### Testing Instructions

* Open user profile and see that the offer location element works the same
* Open React Developer Tools and see that the element is written in React

Continued work on #1019
